### DISCUSSION TITLE
Plugin does not work. please.

### DISCUSSION THREAD

**Discussion by yongam77** [vslinko/obsidian-outliner#329] [Archive]
I have installed the outliner, but when I tried to move lists and sublists up and down on my MacBook, the hotkeys of outliers did not work as intended. Instead of moving lists up or down, it just highlights all the text above or below the selected cursor. Can anyone solve this problem? Please.

**Comment by yongam77** [comment]
I have downgraded the most updated version to v,2.3.4. so the problem has been solved. I think that there is a bug in it.

**Reply by vslinko** [reply]
Yes, there was a bug https://github.com/vslinko/obsidian-outliner/issues/325
Thank you for the feedback.

**Comment by branchm** [comment]
I am seeing the same behavior.  Plugin version 2.3.0.  Obsidian version 0.15.9.  
At first I thought the plugin might not be Enabling when I saw the "not compatible with legacy editor" popup.  But it does appear in fact to be Enabled (I think).  But no gear symbol next to the plugin, so where are all the described settings?  When I look at Setting/HotKeys, I see two entries for "Outline:xxx" but none for "Outliner:xxx".  Basically, it seems like the plugin isn't working even though it is Installed and appears to be Enabled.

**Reply by vslinko** [reply]
@branchm thank you for the feedback. this issue should be fixed in https://github.com/vslinko/obsidian-outliner/releases/tag/2.3.2

**Reply by branchm** [reply]
Thanks.  Upgraded to Plugin 2.4.0.  Now the hotkeys definitely work and I see the plugin options.  
But, I still don't see the vertical lines working as shown in the demo (they are enabled in the settings).  If I put my mouse on the "standard" vertical lines, they don't get larger like in the demo and nothing happens if I click on the (standard) vertical line.

**Reply by vslinko** [reply]
@branchm could you please create an issue about that https://github.com/vslinko/obsidian-outliner/issues/new?assignees=&labels=bug&template=bug_report.md&title=%5BBUG%5D

**Reply by branchm** [reply]
Sorry for the delay.  Issue #348 submitted.  Thanks!
